packages/playground/src/style/border-radius.scss

```diff
@@ -0,0 +1,4 @@
+$border-radius-small: 3px;
```

Comment:
I wouldn't duplicate all these files. If we cannot access them from the playground, that's fine, let's just remove them and not use these variables.
That being said, feel free to just merge this PR and then we can remove them in another PR.

Reply:
Fair point! I think it'd be better to refactor this in a separate PR, not to add even more changes to this one :)